### ISSUE TITLE
rcrack crashes on execution

### ISSUE THREAD

**Issue by antymat** [openaudible/openaudible#196]
While trying to convert AAX to mp3, the rcrack crashes each time. I tun the rcrack from cmd (the same command as logged in the console):

```
rcrack.exe ..\tables  -h d5b6daff0ad8f68066c54546d0f5c65c35a0755e
4 rainbow tables found
memory available: 480850739 bytes
memory for rainbow chain traverse: 160000 bytes per hash, 160000 bytes for 1 hashes
memory for rainbow table buffer: 4 x 21797536 bytes
```
And then this:
![rcrack_has_stopped](https://user-images.githubusercontent.com/2864872/65452660-f2d81a00-de41-11e9-8f3e-444ecab50dc1.png)

In the console itself, I have:
```
Sep 23, 2019 8:39:13 PM org.openaudible.convert.LookupKey lookupKey
SEVERE: Error running rcrack:cmd:"C:\Program Files\OpenAudible\bin\win\rcrack.exe" "C:\Program Files\OpenAudible\bin\tables" -h d5b6daff0ad8f68066c54546d0f5c65c35a0755e 
exit=-1073741819
stdout:
err=
Sep 23, 2019 8:39:13 PM org.openaudible.convert.ConvertJob convert
SEVERE: Error converting book to MP3:Because Internet
java.io.IOException: Error running rcrack. see console.
	at org.openaudible.convert.LookupKey.lookupKey(LookupKey.java:146)
	at org.openaudible.convert.LookupKey.getKeyFromAAX(LookupKey.java:94)
	at org.openaudible.convert.LookupKey.getKeyFromAAX(LookupKey.java:103)
	at org.openaudible.convert.ConvertJob.getActivationBytes(ConvertJob.java:317)
	at org.openaudible.convert.ConvertJob.createMP3(ConvertJob.java:135)
	at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:333)
	at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:384)
	at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:393)

Sep 23, 2019 8:39:13 PM org.openaudible.util.queues.ThreadedQueue jobError
INFO: ConvertQueue error processing convert Because Internet
java.io.IOException: Error running rcrack. see console.
	at org.openaudible.convert.LookupKey.lookupKey(LookupKey.java:146)
	at org.openaudible.convert.LookupKey.getKeyFromAAX(LookupKey.java:94)
	at org.openaudible.convert.LookupKey.getKeyFromAAX(LookupKey.java:103)
	at org.openaudible.convert.ConvertJob.getActivationBytes(ConvertJob.java:317)
	at org.openaudible.convert.ConvertJob.createMP3(ConvertJob.java:135)
	at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:333)
	at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:384)
	at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:393)
```

**Comment by openaudible** [comment]
Check your anti-virus or firewall. 

You should be able to verify the file hasn't changed since it was posted: https://github.com/openaudible/openaudible/blob/master/bin/win_rcrack.exe

**Comment by antymat** [comment]
I switched off the firewall, and the antivirus did not complain about anything. The MD5 of the binary is the same. 
Still crashes.

**Comment by nearlymad** [comment]
I have posted this issue again. I have the same problem.

**Comment by openaudible** [comment]
This may be fixed in 2.0.6.
